packages/cli/src/controllers/me.controller.ts

```diff
@@ -176,7 +176,7 @@ export class MeController {
 			throw new BadRequestError('Personalization answers are mandatory');
 		}
 
-		await this.userRepository.save(
+		await this.authUserRepository.save(
```

Comment:
Should we use the regular `userRepository` when we are updating other than the auth properties?